### DISCUSSION TITLE
Markdown list items overlap each other

### DISCUSSION THREAD

**Discussion by KostasFeg** [remarkjs/remark#596] [Q&A] [answered]
Hi guys, I'm having this weird issue and I don't get what's going on.
![Στιγμιότυπο οθόνης 2020-12-30 110742](https://user-images.githubusercontent.com/56806349/103341402-852c8c00-4a8f-11eb-8c48-3c604abfd1b1.png)
![Στιγμιότυπο οθόνης 2020-12-30 110816](https://user-images.githubusercontent.com/56806349/103341404-85c52280-4a8f-11eb-8c17-4a7aa69c4e97.png)

The first image is on a textarea that I use plain text and the second one is inside the reactmarkdown. 
Can anyone guess what's going on here?

**Comment by wooorm** [comment, marked as answer]
Looks like you have CSS that’s messing it up.

**Reply by KostasFeg** [reply]
Yep, that's the issue for sure, removed the CSS files and it displays the markdown properly, now I'm gonna try find the specific issue, thanks!
